lite_mode/lite_internal_deploy.py

```diff
@@ -0,0 +1,96 @@
+"""
+This module introduces `LiteInternalDeploy`, optimized lite-mode version of InternalDeploy.
+"""
+from typing import List
+
+from starkware.starknet.business_logic.transaction.objects import InternalDeploy
+from starkware.starknet.services.api.gateway.transaction import (
+    Deploy,
+    Transaction,
+    EverestTransaction,
+)
+from starkware.starknet.services.api.contract_class import ContractClass
+from starkware.starknet.business_logic.utils import verify_version
+from starkware.starknet.core.os.contract_address.contract_address import (
+    calculate_contract_address_from_hash,
+)
+from starkware.starknet.core.os.class_hash import compute_class_hash
+from starkware.starknet.business_logic.transaction.objects import InternalTransaction
+
+from starknet_devnet.util import to_bytes
```

Comment:
This shouldn't be imported from `starknet_devnet.util`, look into that file to see from where it imports.

Reply:
Ok, I didn't realize there were two functions, as discussed separately, we should try using just the imported one.